openjfx/src/main/java/science/aist/imaging/service/openjfx/imageprocessing/wrapper/WritableImageWrapper.java

```diff
@@ -0,0 +1,94 @@
+package science.aist.imaging.service.openjfx.imageprocessing.wrapper;
+
+import javafx.scene.image.PixelReader;
+import javafx.scene.image.PixelWriter;
+import javafx.scene.image.WritableImage;
+import javafx.scene.paint.Color;
+import science.aist.imaging.api.domain.wrapper.AbstractImageWrapper;
+import science.aist.imaging.api.domain.wrapper.ChannelType;
+
+/**
+ * <p>Created by Christoph Praschl on 16.06.2021</p>
+ * <p>ImageWrapper implementation for JavaFX's WritableImage class</p>
+ *
+ * @author Christoph Praschl christoph.praschl@fh-hagenberg.at
```

Comment:
Please remove created by and emails address, and add a `@since` annotation. Do not forget to also add it to the changelog.md then. I guess it should be version: 1.2.0?